core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileOutputWriter.scala

```diff
@@ -51,27 +51,43 @@ class ProfileOutputWriter(outputDir: String, filePrefix: String, numOutputRows:
     }
   }
 
+  def write(headerText: String, outRows: Seq[ProfileResult],
+      emptyTableText: Option[String] = None, tableDesc: Option[String] = None): Unit = {
+    writeTextTable(headerText, outRows, emptyTableText, tableDesc)
+    ProfileOutputWriter.writeCSVTable(headerText, outRows, outputCSV, outputDir)
```

Comment:
Should check if `outputCSV` before calling this and the `writeCSVTable` should not be taking an argument `outputCSV`
```
if (outputCSV) {
  ProfileOutputWriter.writeCSVTable(headerText, outRows, outputDir)
}
```

Reply:
added this check before calling `ProfileOutputWriter.writeCSVTable`